com.unity.render-pipelines.core/Runtime/Debugging/ShaderDebugPrintManager.cs

```diff
@@ -0,0 +1,323 @@
+using System;
+using System.Collections.Generic;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+using UnityEngine.Assertions;
+
+namespace UnityEngine.Rendering
+{
+    public sealed class ShaderDebugPrintManager
+    {
+        private static readonly ShaderDebugPrintManager instance = new ShaderDebugPrintManager();
+
+        private const int DebugUAVSlot = 7;
+        private const int FramesInFlight = 4;
+        private const int MaxBufferElements = 1024 * 16; // Must match the shader size definition
+
+        private List<GraphicsBuffer> m_OutputBuffers = new List<GraphicsBuffer>();
+
+        private List<Rendering.AsyncGPUReadbackRequest> m_readbackRequests =
```

Comment:
```suggestion
        private List<Rendering.AsyncGPUReadbackRequest> m_ReadbackRequests =
```
Naming private variable with our coding standards